website/docs/podman/gpu.md

```diff
@@ -159,9 +161,11 @@ RUN dnf -y install dnf-plugins-core && \
 3. Verify you can see the GPU by running a test container:
 
 ```sh
-$  podman run --rm -it --device /dev/dri --name gpu-info quay.io/slopezpa/fedora-vgpu vulkaninfo | grep "GPU"
+$  podman run --rm -it --device /dev/dri --name gpu-info <gpu-container-name>  vulkaninfo | grep "GPU"
 ```
 
+- Replace `gpu-container-name` with a valid container name, such as `quay.io/slopezpa/fedora-vgpu`.
```

Comment:
Also incorrect, you are supplying name with `--name gpu-info` not `gpu-container-name`. 

I also feel that you do not need to add this comment? Since we are building it and supplying it a name anyways.

Reply:
Removed it.